lib/membrane_http_adaptive_stream/sink_bin.ex

```diff
@@ -114,7 +114,10 @@ defmodule Membrane.HTTPAdaptiveStream.SinkBin do
     state = %{
       muxer_segment_duration: opts.muxer_segment_duration,
       mode: opts.hls_mode,
-      streams_counter: 0
+      streams_to_start: 0,
+      started?: false,
+      streams_to_end: 0,
+      ended?: false
```

Comment:
maybe `status: :awaiting_start | :playing | :ended` instead of `started?` and `ended?`?

Reply:
yup, much better